tests/testsuite/profile_targets.rs

```diff
@@ -607,14 +607,13 @@ fn profile_selection_check_all_targets_test() {
 [COMPILING] foo [..]
 [RUNNING] `rustc --crate-name build_script_build build.rs [..]--crate-type bin --emit=dep-info,link -C codegen-units=1 -C debuginfo=2 [..]
 [RUNNING] `[..]target/debug/build/foo-[..]/build-script-build`
-foo custom build PROFILE=debug DEBUG=true OPT_LEVEL=0
+[foo 0.0.1] foo custom build PROFILE=debug DEBUG=true OPT_LEVEL=0
 [RUNNING] `rustc --crate-name foo src/lib.rs [..]--crate-type lib --emit=dep-info,metadata -C codegen-units=1 -C debuginfo=2 [..]
 [RUNNING] `rustc --crate-name foo src/lib.rs [..]--emit=dep-info,metadata -C codegen-units=1 -C debuginfo=2 --test [..]
 [RUNNING] `rustc --crate-name test1 tests/test1.rs [..]--emit=dep-info,metadata -C codegen-units=1 -C debuginfo=2 --test [..]
 [RUNNING] `rustc --crate-name foo src/main.rs [..]--emit=dep-info,metadata -C codegen-units=1 -C debuginfo=2 --test [..]
 [RUNNING] `rustc --crate-name bench1 benches/bench1.rs [..]--emit=dep-info,metadata -C codegen-units=1 -C debuginfo=2 --test [..]
 [RUNNING] `rustc --crate-name ex1 examples/ex1.rs [..]--emit=dep-info,metadata -C codegen-units=1 -C debuginfo=2 --test [..]
-[FINISHED] dev [unoptimized + debuginfo] [..]
```

Comment:
I don't think this line should have been removed.

Reply:
Oops, that was a rebase mistake! I swear I ran tests locally, I'm not sure why it didn't fail here.

Reply:
Oh, this test only runs on nightly and I was using stable cargo to run `cargo test` locally:
https://github.com/rust-lang/cargo/pull/6164/commits/20920c01315003e5bc34ff0ca41d55a44e6ec07e#diff-2cbe411b955101d5ff2d27d7a2b63503R571